widgets/post-carousel/post-carousel.php

```diff
@@ -156,6 +156,33 @@ function get_widget_form(){
 				'label' => __('Posts query', 'so-widgets-bundle'),
 				'hide' => true,
 			),
+
+			'design' => array(
+				'type' => 'section',
+				'label' => __( 'Design', 'so-widgets-bundle' ),
+				'hide' => true,
+				'fields' => array(
+					'navigation_color' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation arrow color', 'so-widgets-bundle' ),
+						'default' => '#fff',
+					),
+					'navigation_color_hover' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation arrow color hover', 'so-widgets-bundle' ),
+					),
+					'navigation_background' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation background', 'so-widgets-bundle' ),
+						'default' => '#333',
+					),
+					'navigation_background_hover' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation background hover', 'so-widgets-bundle' ),
```

Comment:
Let's perhaps label this "Navigation hover background". Unless there is precedence I've missed.